<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForumPage</class>
 <widget class="QFrame" name="ForumPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QListWidget" name="forumListWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addQuestionButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>вопрос</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="newQuestionTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addAnswerButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ответ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="newAnswerTextEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
